src/netex-convertor/netexGenerator.ts

```diff
@@ -271,15 +280,110 @@ const netexGenerator = (
     const updatePriceFareFrame = (priceFareFrame: NetexObject): NetexObject => {
         const priceFareFrameToUpdate = { ...priceFareFrame };
 
-        priceFareFrameToUpdate.id = `epd:UK:${coreData.operatorIdentifier}:FareFrame_UK_PI_FARE_PRODUCT:${coreData.placeholderGroupOfProductsName}@pass:op`;
+        const ticketIdentifier: string = isPointToPointTicket(ticket)
+            ? coreData.lineIdName
+            : coreData.placeholderGroupOfProductsName;
+
+        priceFareFrameToUpdate.id = `epd:UK:${coreData.operatorIdentifier}:FareFrame_UK_PI_FARE_PRODUCT:${ticketIdentifier}@pass:op`;
+        priceFareFrameToUpdate.tariffs.Tariff.id = isPointToPointTicket(ticket)
+            ? `Tariff@${coreData.ticketType}@${coreData.lineIdName}`
+            : `op:Tariff@${coreData.placeholderGroupOfProductsName}`;
+        priceFareFrameToUpdate.tariffs.Tariff.Name.$t = `${ticketIdentifier} - Fares for ${coreData.ticketType} ticket`;
```

Comment:
For a pointToPoint ticket, this used to be `${operatorPublicNameLineNameFormat} - ${matchingData.type} fares`. Does this `Tariff.Name` not still want to include this `operatorPublicNameLineNameFormat`?

Reply:
We also used to have a `Tariff.Description` in the old periodTicketNetexGenerator. I guess we could use this for more detail instead of lumping it all in the `Tariff.Name` if we wanted?